data/hai/hai outfits.txt

```diff
@@ -471,6 +472,17 @@ outfit "Hai Williwaw Cooling"
 	"cooling" 11
 	description "While Hai technology is fairly heat efficient, occasionally they outfit their ships such that they sporadically overheat and shut down. When that happens, they install these."
 
+outfit "Hai Zephyr Cooling"
+	category "Systems"
+	cost 72000
+	thumbnail "outfit/hai zephyr"
+	"mass" 14
+	"outfit space" -14
+	"cooling" 24
+	description `This more advanced cooling system has been developed by the Unfettered with significant help from Korath scientists, who were highly amused with the Williwaw's simultaneous "primitive" design and similarities to a Korath heat shunt.`
```

Comment:
Both over simultaneous works a bit better, but the other changes aren't appropriate.

Adding "its" also requires a comma before the and, since it changes it into a separate complete clause.

Reply:
Sorry, but that's incorrect.

> its similarity to a Korath heat shunt

is not an independent clause, so a comma would not be appropriate.

Reply:
```suggestion
	description `This state-of-the-art cooling system has been developed by the Unfettered with significant help from Korath scientists, who were highly amused with both the Williwaw's "primitive" design and its similarity to a Korath heat shunt.`
```

Reply:
Full disclosure, I think read this as "it's" somehow (though that would still not be a -correct- sentence without other changes)? You're correct on that.